src/Sylius/Bundle/OrderBundle/Resources/config/services.xml

```diff
@@ -26,6 +26,9 @@
         <parameter key="sylius.order_item_quantity_modifier.class">Sylius\Bundle\OrderBundle\Modifier\OrderItemQuantityModifier</parameter>
         <parameter key="sylius.form.data_mapper.property_path.class">Symfony\Component\Form\Extension\Core\DataMapper\PropertyPathMapper</parameter>
         <parameter key="sylius.form.data_mapper.order_item_quantity.class">Sylius\Bundle\OrderBundle\Form\DataMapper\OrderItemQuantityDataMapper</parameter>
+
+        <parameter key="sylius.aggregator.adjustments.class">Sylius\Bundle\OrderBundle\Aggregator\AdjustmentsByLabelAggregator</parameter>
```

Comment:
We do not have any other aggregators and this is not a generic concept for all resources, etc. I'd say here ``sylius.adjustments_aggregator``.